### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Add Travis CI build service. The build log is available [here](https://travis-ci.org/peter279k/php-reflect/builds/522420372).

- Set the different PHPUnit versions to support different PHP versions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "issues": "https://github.com/llaville/php-reflect/issues"
     },
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^5.6|^7.0",
```
Comment (peter279k) [inline composer.json]: The `doctrine/collections` `^1.4` version needs `php-5.6` version at least and let this package require `php-5.6` version at least.

File: tests/bootstrap.php [hunk 2]
```diff
@@ -1,18 +1,3 @@
...
-            => __DIR__ . '/Analyser/FooAnalyser.php',
-        'Bartlett\Tests\Reflect\Analyser\BarAnalyser'
-            => __DIR__ . '/Analyser/BarAnalyser.php',
-        'Bartlett\Tests\Reflect\Model\GenericModelTest'
-            => __DIR__ . '/Model/GenericModelTest.php',
-        'Bartlett\Tests\Reflect\Environment\YourLogger'
-            => __DIR__ . '/Environment/YourLogger.php',
-    )
-);
+require_once __DIR__ . '/../vendor/autoload.php';
```
Comment (peter279k) [inline tests/bootstrap.php]: Using the `composer` autoloader can load the all classes including test classes automatically.

File: composer.json (hunk 1 above)
Comment (llaville) [reply composer.json]: Hello. 
Thanks for your contribution, but perhaps you should have noticed that I'm working on a new master branch 5.0 that will drop support to PHP 5.

**Comment by peter279k** [comment]
Hi @llaville, thank you for your reply.

I've updated the package require `php-7.x` versions at least :).

**Comment by llaville** [comment]
Hello Chun-Sheng
First, I would like to thanks you for your contribution. But i'd like to add Github Actions for CI !
It's on my TODO list, but i've other works to do before.
If you think you can contribute in this way, by a new PR. I promise to check it ASAP !
Sorry for answer delayed.

**Comment by peter279k** [comment]
@llaville, no problem.

I will create another PR to migrate to GitHub Action :).

**Pull request closed without merging**